docs/user-guide/quick-start/contributor-api.md

```diff
@@ -265,19 +309,24 @@ to study and link data to samples.
     2. Click on the endpoint  to **Create a link between a group of expression objects and a group of 
    samples objects**: 
    `/api/v1/as-curator/integration/link/expression/group/{sourceId}/to/sample/group/{targetId}`
-   ![Data Integration](quick-start-images/post-data-integration.png)
+  
+    ![Data Integration](quick-start-images/post-data-integration.png)
+    <figcaption>Click on Expression integration as Curator to find the endpoints to link experimental data with samples (and the study). Select the endpoint ==/api/v1/as-curator/integration/link/expression/group/{sourceId}/to/sample/group/{targetId}==</figcaption>
```

Comment:
Could we please here and in general everywhere specify what type of endpoint do we need? because when we mention only its "name", e.g. /api/v1/as-curator/integration/link/expression/group/{sourceId}/to/sample/group/{targetId} - it's not enough, because with this name we have Post endpoint and also DELETE endpoint, it's better to mention also what type of endpoint should be used, not just mention its name